Bottom: f1f3fc67f8077810a8177706d399d99f4e43f425
Top:    aa5a2df45a24a02c51a1cb9aff28354d4f91379c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 11:56:32 -0400

Refresh of dspeak-audio

---

diff --git a/sound/soc/fsl/dspeak01_fabric.c b/sound/soc/fsl/dspeak01_fabric.c
index bd35a14..c6c7890 100644
--- a/sound/soc/fsl/dspeak01_fabric.c
+++ b/sound/soc/fsl/dspeak01_fabric.c
@@ -113,7 +113,7 @@ static int __devinit dspeak01_fabric_probe(struct of_device *op,
 	if (!fabric.clock)
 		return -ENODEV;
 
-	of_snd_soc_register_machine("DSPEAK01", &dspeak01_fabric_ops);
+	of_snd_soc_register_fabric("DSPEAK01", &dspeak01_fabric_ops, NULL);
 	return 0;
 }
